docs/maps/index.asciidoc

```diff
@@ -11,17 +11,43 @@ With *Elastic Maps*, you can:
 
 * Create maps with multiple layers and indices.
 * Upload GeoJSON files into Elasticsearch.
-* Embed your map in Dashboards.
-* Plot individual documents or use aggregations to plot any data set, no matter how large.
-* Create choropleth maps.
-* Use data driven styling to symbolize features from property values.
+* Embed your map in dashboards.
+* Use data driven styling to symbolize features from data values.
 * Focus the data you want to display with searches.
 
-Start your tour of *Elastic Maps* with the <<maps-getting-started, getting started tutorial>>.
+*Ready to get started?* Start your tour of *Elastic Maps* with the <<maps-getting-started, getting started tutorial>>.
+
+[float]
+=== Create maps with multiple layers and indices
+You can use multiple layers and indices to show all your data in a single map. This enables your map to show how data sits relative to physical features like weather patterns, human-made features like international borders, and business-specific features like sales regions. You can plot individual documents or use aggregations to play any data set, no matter how large.
 
 [role="screenshot"]
 image::maps/images/sample_data_ecommerce.png[]
 
+[float]
+=== Upload GeoJSON files into Elasticsearch
+Elastic Maps makes it easy to import geospatial data into the Elastic Stack. Using the GeoJSON Upload feature, you can drag and drop your point and shape data files directly into Elasticsearch, and then use them as layers in the map.
+
+[float]
+=== Embed your map in dashboards
+Viewing data from different angles provides better insights. Dimensions that are obscured in one visualization might be illuminated in another. Add your map to a dashboard and view your geospatial data alongside bar charts, pie charts, tag clouds, and more.
+
+In the following example, a choropleth map showing San Diego non-emergency service request density by council district is embedded in a dashboard to better understand when services are requested and gain insight into the top requested services.
+
+[role="screenshot"]
+image::maps/images/embed_in_dashboard.jpeg[]
+
+[float]
+=== Use data driven styling to symbolize features from data values
+You can customize each layer to highlight meaningful dimensions in your data. For example, you can use dark colors to symbolize areas with more web log traffic, and lighter colors to symbolize areas with less traffic. .
+
+[float]
+=== Focus the data you want to display with searches
+Use the search bar for real-time search for all of your Elasticsearch layers. Draw a polygon on the map or use the shape from features to create spatial filters to narrow search results to documents that either intersect with, are within, or do not intersect with the specified geometry. Filter individual layers to compares facets.
+
```

Comment:
Let's talk about the last two sections.  I'd like to see descriptions tie better to the titles.